src/Internal.AspNetCore.Sdk/build/Common.props

```diff
@@ -22,6 +22,8 @@ Usage: this should be imported once via NuGet at the top of the file.
     <PackageProjectUrl>https://asp.net</PackageProjectUrl>
     <PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
     <Serviceable Condition="'$(Configuration)' == 'Release'">true</Serviceable>
+    <DebugType>Portable</DebugType>
+    <SourceLinkCreate>true</SourceLinkCreate>
```

Comment:
Does anyone know a good way to tell if a project is a src project at this point? Right now we're doing SourceLink for test projects, which shouldn't HURT anything, but is kind of a waste.

Reply:
Move this to Common.targets so this is added after the csproj is evaluated, and condition on IsPackable.
```xml
<SourceLinkCreate Condition="'$(SourceLinkCreate )' == '' AND  '$(IsPackable)' == 'true'">true</SourceLinkCreate>
```

Reply:
As discussed offline, if we put it in Common.targets it doesn't get executed before we compile, and therefore the json doesn't end up in the PDBs.